src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package wolox.training.models;
+import java.util.ArrayList;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import wolox.training.errors.book.*;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+@Table(name="users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USERS_SEQ")
+    @SequenceGenerator(name = "USERS_SEQ", sequenceName = "USERS_SEQ")
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthday;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
```

Comment:
The many to many relationship should work without the @JoinTable annotation since by default it defines the same thing that you have put.

Reply:
Okay, this is okay